pkg/cortex/cortex_test.go

```diff
@@ -92,10 +94,23 @@ func TestCortex(t *testing.T) {
 				Directory: os.TempDir(),
 			},
 		},
-
-		Target: []string{All, Compactor},
+		AlertmanagerStorage: alertstore.Config{
+			Config: bucket.Config{
+				Backend: "local",
+			},
+		},
+		Target: []string{All},
 	}
 
+	externalURL := flagext.URLValue{}
+	err := externalURL.Set("http://localhost/api/prom")
```

Comment:
http://localhost/api/prom
```suggestion
	err := externalURL.Set("http://localhost/alertmanager")
```
it's more appropiate